VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/NotificationsSettingsScreen/NotificationsSettingsScreen.test.tsx

```diff
@@ -52,79 +37,125 @@ context('NotificationsSettingsScreen', () => {
     notificationsEnabled: boolean,
     systemNotificationsOn: boolean,
     preferences: PushPreference[],
-    errorsState: ErrorsState = initialErrorsState,
   ) => {
     const props = mockNavProps()
     mockPushEnabled = notificationsEnabled
 
-    render(<NotificationsSettingsScreen {...props} />, {
-      preloadedState: {
-        ...InitialState,
-        notifications: {
-          ...InitialState.notifications,
-          preferences,
-          systemNotificationsOn,
+    const notificationQueriesData: QueriesData = [
+      {
+        queryKey: notificationKeys.settings,
+        data: {
+          preferences: preferences,
+          systemNotificationsOn: systemNotificationsOn,
+          deviceToken: '1',
+          initialUrl: '1',
+          tappedForegroundNotification: false,
         },
-        errors: errorsState,
       },
-    })
+    ]
+
+    render(<NotificationsSettingsScreen {...props} />, { queriesData: notificationQueriesData })
   }
 
   describe('appointment reminders switch', () => {
-    it('value should be true when pref is set to true', () => {
-      renderWithProps(false, true, [apptPrefOn])
-      expect(screen.getByRole('switch', { name: 'Upcoming appointments' }).props.accessibilityState.checked).toEqual(
-        true,
+    it('value should be true when pref is set to true', async () => {
+      const prefMock = AsyncStorage.getItem as jest.Mock
+      when(prefMock).calledWith('@store_device_endpoint_sid').mockResolvedValue('1')
+
+      const responseData: GetPushPrefsResponse = {
+        data: {
+          type: 'string',
+          id: 'string',
+          attributes: {
+            preferences: [apptPrefOn],
+          },
+        },
+      }
+      when(api.get as jest.Mock)
+        .calledWith(`/v0/push/prefs/1`)
```

Comment:
```suggestion
        .calledWith('/v0/push/prefs/1')
```
Template literals aren't needed here

Reply:
updated